models/iaf_cond_beta.h

```diff
@@ -0,0 +1,428 @@
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef IAF_COND_BETA_H
+#define IAF_COND_BETA_H
+
+// Generated includes:
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// C includes:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+// Includes from nestkernel:
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "recordables_map.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+/* BeginDocumentation
+Name: iaf_cond_beta - Simple conductance based leaky integrate-and-fire neuron
+                       model.
+
+Description:
+iaf_cond_beta is an implementation of a spiking neuron using IAF dynamics with
+conductance-based synapses. Incoming spike events induce a post-synaptic change
+of conductance modelled by an beta function. The beta function
+is normalised such that an event of weight 1.0 results in a peak current of 1 nS
+at t = tau_syn.
+
+Parameters:
+The following parameters can be set in the status dictionary.
+
+V_m          double - Membrane potential in mV
+E_L          double - Leak reversal potential in mV.
+C_m          double - Capacity of the membrane in pF
+t_ref        double - Duration of refractory period in ms.
+V_th         double - Spike threshold in mV.
+V_reset      double - Reset potential of the membrane in mV.
+E_ex         double - Excitatory reversal potential in mV.
+E_in         double - Inhibitory reversal potential in mV.
+g_L          double - Leak conductance in nS;
+tau_rise_ex  double - Rise time of the excitatory synaptic beta function in ms.
+tau_decay_ex double - Rise time of the excitatory synaptic beta function in ms.
+tau_rise_in  double - Rise time of the inhibitory synaptic beta function in ms.
+tau_decay_in double - Rise time of the inhibitory synaptic beta function in ms.
+I_e          double - Constant input current in pA.
+G_ex        double - Constant external input conductance in uS.
+G_in        double - Constant external input conductance in uS.
+
+Sends: SpikeEvent
+
+Receives: SpikeEvent, CurrentEvent, DataLoggingRequest
+
+References:
+
+Meffin, H., Burkitt, A. N., & Grayden, D. B. (2004). An analytical
+model for the large, fluctuating synaptic conductance state typical of
+neocortical neurons in vivo. J.  Comput. Neurosci., 16, 159-175.
+
+Bernander, O ., Douglas, R. J., Martin, K. A. C., & Koch, C. (1991).
+Synaptic background activity influences spatiotemporal integration in
+single pyramidal cells.  Proc. Natl. Acad. Sci. USA, 88(24),
+11569-11573.
+
+Kuhn, Aertsen, Rotter (2004) Neuronal Integration of Synaptic Input in
+the Fluctuation- Driven Regime. Jneurosci 24(10) 2345-2356
+
+Rotter & Diesmann, Biol Cybern 81:381 (1999)
+
+Roth and van Rossum,
+Ch 6, in De Schutter, Computational Modeling Methods for Neuroscientists,
+MIT Press, 2010.
+
+
+Author: Schrader, Plesser
```

Comment:
Please put your name here instead. (I suppose you modified a model that was written by Schrader, Plesser?)